### DISCUSSION TITLE
America TV [PY]

### DISCUSSION THREAD

**Discussion by thelmagali** [iptv-org/community#457] [Show and tell]
### Channel ID

AmericaParaguay.py

### Stream URL (optional)

https://live-ak.vimeocdn.com/exp=1686948139~acl=%2F2717b064-8f04-4511-8fde-5a14cfdc2b7d%2F%2A~hmac=8fc3c91ee33e26704791436d6ba837f872767fe5ec27566a8de0a41d231606ab/2717b064-8f04-4511-8fde-5a14cfdc2b7d/chunklist_b3256000.m3u8?max_segments=10

### Notes (optional)

Taken from their Vimeo channel: https://vimeo.com/836771615
Channel website: https://www.grupoamerica.info/en-vivo

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://live-ak.vimeocdn.com/exp=1686948139~acl=%2F2717b064-8f04-4511-8fde-5a14cfdc2b7d%2F%2A~hmac=8fc3c91ee33e26704791436d6ba837f872767fe5ec27566a8de0a41d231606ab/2717b064-8f04-4511-8fde-5a14cfdc2b7d/chunklist_b3256000.m3u8?max_segments=10 (Offline)
<img width="840" alt="image" src="https://github.com/iptv-org/iptv/assets/7253922/891ebc05-82d0-4f8c-9682-939f4ea5f8f0">

**Comment by BellezaEmporium** [comment]
inside https://live-api.vimeocdn.com/sessions/b72464e8-fcc6-4228-a95e-a00a90a65f07/playlist/hls.m3u8?cdn=akamai&json=1&~exp=1687039200&~id=player&~sig=MEVI2NS8TIfmSgbck47DHuSvXeD7qWa0INhN3XtkoAQ (signature changes)

**Comment by Sphinxroot** [comment]
@BellezaEmporium  does your online  streamlinker handle this https://vimeo.com/836771615 ?

**Comment by BellezaEmporium** [comment]
> @BellezaEmporium  does your online  streamlinker handle this https://vimeo.com/836771615 ?

It's having issues with Vimeo links, this one included.
